conda_smithy/templates/run_docker_build.tmpl

```diff
@@ -26,10 +26,12 @@ show_channel_urls: true
 CONDARC
 )
 
+HOST_USER_ID=$( hash docker-machine 2>/dev/null && docker-machine ssh $(docker-machine active) 'id -u $(whoami)' || id -u $(whoami) )
```

Comment:
No problem.
I tried to test it on Linux. I don't if I'm doing something wrong, but the ```docker-machine``` does not mount automatically my home dir. So the ```docker``` inside ```docker-machine``` just mount an empty dir in the container. I think it is probably because VirtualBox auto-mount feature is not working.

I could mount it manually, but I would like to do this as close as possible to what happens on Mac. Could you please show me the output of ```docker-machine ssh mount``` and ```docker-machine ssh ls -la /home``` ?